### PR TITLE
Contributing book bump

### PR DESCRIPTION
**Pull request by SwayStar123**
## Type of change

- Improvement (refactoring, restructuring repository, cleaning tech debt, ...)

## Changes

The following changes have been made:

- Forc version updated to 0.46.0

### CODE REVIEW

File: .docs/contributing-book/src/code/fuel-toolchain.toml [hunk 1]
```diff
@@ -2,5 +2,5 @@
 channel = "latest-2023-08-16"
```
Comment (K1-R1) [inline .docs/contributing-book/src/code/fuel-toolchain.toml:2]: The channel should be updated too.

**Review by K1-R1 — COMMENTED** [review]
You may have to update the ci for this to work to. You can use the previous bumping of the contributing book as a reference: https://github.com/FuelLabs/sway-applications/commit/3c591c1be8ae2c8d9613a7b9eb6ecdb44566dd01

**Comment by SwayStar123** [comment]
> You may have to update the ci for this to work to. You can use the previous bumping of the contributing book as a reference: [3c591c1](https://github.com/FuelLabs/sway-applications/commit/3c591c1be8ae2c8d9613a7b9eb6ecdb44566dd01)

Well, i didnt do that on purpose because the channels seem to be broken, the latest one is more than 3 weeks behind, and so still doesnt have version 0.46.0

**Comment by SwayStar123** [comment]
> > Well, i didnt do that on purpose because the channels seem to be broken, the latest one is more than 3 weeks behind, and so still doesnt have version 0.46.0
> 
> hmmm. We may just have to wait to merge any bumps until the channels are fixed. I suppose we could work off of the nightly channel until then, and merge once the latest channel is fixed, what do you think?

I dont need the cis to pass to do the actual update, so i can just update all the apps while waiting for the fix

**Comment by bitzoic** [comment]
Closing in favor of https://github.com/FuelLabs/sway-applications/pull/765

**Pull request closed without merging**